Jin

uzzzzin opened the pull request.

Commit:
#1 회원 추가, 회원 출력 기능 구현
회원 추가 기능 중, 회원 등록 후 또 회원 추가 등록을 할건지에 대한 부분은 제외하고 구현.

jeontigger commented:
LGTM
cin 자료형 잘못 받았을 때 예외처리법!
```c++
cin.clear(); // cin의 에러비트를 클리어해주는 함수였음! 자료형 잘못 받았으니까 에러 비트가 켜졌을 것이고
cin.ignore(256(대충 큰 숫자), '\n'(기준이되는 char));  //두번째 파라미터인 char 만나기 전까지의 버퍼 내용을 버려버림
```
아래는 참고 레퍼런스 페이지
https://cplusplus.com/reference/istream/istream/ignore/

Commit:
#1 초기화면으로 돌아가기 위한 기능 구현
2주차의 기능 추가를 생각해 AlwaysLast() 함수로 코드를 분리(?) 시켰고, 추가 회원 등록을 진행할 때는 이 함수를 변형해서 코드 작성함.

jeontigger commented on Jin/UserInfoAdmin.cpp:
```diff
@@ -106,12 +93,27 @@ void UserInfoAdmin::PrintUserInfo()
+{
+	cout << " 0이 아닌 숫자를 입력하면 초기화면으로 돌아갑니다. " << endl << " 0이 아닌 숫자를 입력하세요.";
+	int next;
+	cin >> next; // 문자를 받아버리면 0으로 치부하고 while문에 들어옴.
+	cin.clear();
+	cin.ignore(100, '\n');
+	while (0 == next)
+	{
+		cout << " 잘못된 입력입니다. 0이 아닌 숫자를 입력하세요.";
+		cin >> next;
+		cin.clear();
+		cin.ignore(100, '\n');
+	}
```
반복문 이외에 반복되는 구조

jeontigger reviewed (commented):
AlwaysLast에서 0을 입력하면 어떻게 되는지?
AlwaysLast의 기능과 함수명이 매치되도록 재고려하면 좋을듯!
